client/galaxy/scripts/viz/circster.js

```diff
@@ -634,10 +634,10 @@ var CircsterTrackView = Backbone.View.extend({
             track.get('config').set_value('max_value', self.data_bounds[1], {silent: true});
 
             // Merge chroms layout with data.
-            layout_and_data = _.zip(chrom_arcs, genome_wide_data),
+            var layout_and_data = _.zip(chrom_arcs, genome_wide_data);
 
             // Render each chromosome's data.
-            chroms_data_layout = _.map(layout_and_data, function(chrom_info) {
+            _.each(layout_and_data, function(chrom_info) {
```

Comment:
Huh, yeah, I made the bad assumption that map() was intentional and that we needed that array later, but it definitely doesn't look like as I dig a little more.  This is indeed probably better as `_.each`.